guide/deployment/docker/build-a-dockerfile.md

```diff
@@ -0,0 +1,89 @@
+# Building your Docker image
+
+<Definition content="A Docker image is a read-only template containing instructions for creating a container that can run on the Docker platform. It provides an easy way to package up applications and pre-configured server environments that you can use privately or publicly with other Docker users." />
```

Comment:
Can't Definition just be removed all together and TIP be used here

Reply:
Yep, https://github.com/discordjs/guide/pull/1047/commits/0b15fb4ff259ec40e787403811ade972834cf269